src/Algebra/Properties/Magma.agda

```diff
@@ -0,0 +1,75 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some items and theory for Magma
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Algebra using (Magma)
+open import Algebra.Definitions using (Commutative; LeftQuotient)
+open import Data.Product using (_×_; _,_)
+open import Function using (_∘_)
+open import Level using (_⊔_)
+open import Relation.Binary using (Rel; _Respects_; _Respects₂_)
+open import Relation.Nullary using (¬_)
+
+module Algebra.Properties.Magma {a ℓ} (M : Magma a ℓ) where
+
+open Magma M
+
+------------------------------------------------------------------------------
+-- The divisibility relation
+------------------------------------------------------------------------------
+
+infix 5 _∣_ _∤_ _∣∣_ _¬∣∣_
+
+private C = Carrier
```

Comment:
You should do this via renaming rather than a private definition. It's also more standard to call it A rather than C.

Reply:
Done.